www/js/diary/list.js

```diff
@@ -277,14 +273,44 @@ angular.module('emission.main.diary.list',['ui-leaflet',
             DiaryHelper.directiveForTrip);
           Timeline.setTripWrappers(currDayTripWrappers);
 
+          let tripFromNotification = null;
+
           $scope.data.currDayTripWrappers.forEach(function(tripgj, index, array) {
             $scope.populateModeFromTimeline(tripgj, $scope.data.unifiedConfirmsResults.modes);
             $scope.populatePurposeFromTimeline(tripgj, $scope.data.unifiedConfirmsResults.purposes);
             $scope.populateSurveyAnswerFromTimeline(tripgj, $scope.data.unifiedConfirmsResults.surveyAnswers);
             $scope.populateBasicClasses(tripgj);
             $scope.populateCommonInfo(tripgj);
           });
+          if($rootScope.displayingIncident == true && $rootScope.notificationData) {
+            const matchingTrips = $scope.data.currDayTripWrappers.filter(trip => {
+              const matchingInput = ConfirmHelper.getUserInputForTrip(trip, [{
+                data: {
+                  start_ts: $rootScope.notificationData.start_ts,
+                  end_ts: $rootScope.notificationData.end_ts,
+                },
+                metadata: {
+                  time_zone: 'Asia/Bangkok'
```

Comment:
you need to use the current timezone instead of the hardcoded value.

Reply:
@atton16 are you going to resolve this? You can use the arrow library to get this information.

Reply:
done